app/jobs/concerns/resource_request_jobs/derivativo_job_behaviors.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ResourceRequestJobs
+  module DerivativoJobBehaviors
+    extend ActiveSupport::Concern
+
+    class_methods do
+      def create_callback(resource_request, _digital_object)
```

Comment:
Sorry to be so picky about naming, but when I started reading this method I thought it might be a factory method that returns return a proc (i.e. I thought it would "create a callback").